addons/mass_mailing_event_track/models/event.py

```diff
@@ -16,7 +16,7 @@ def action_mass_mailing_track_speakers(self):
             target='current',
             context=dict(
                 default_mailing_model_id=self.env.ref('website_event_track.model_event_track').id,
-                default_mailing_domain="[('event_id', 'in', %s), ('stage_id.is_cancel', '!=', True)]" % self.ids,
+                default_mailing_domain=repr([('event_id', 'in', self.ids), ('stage_id.is_cancel', '!=', True)]),
```

Comment:
"repr" will evaluate to a string containing "self.ids" right?
I think we specifically want `"[('event_id', 'in', [1,2,3])]"` here and not `"[('event_id', 'in', self.ids)]"`.

Eeeeeeeh not sure 🤷‍♂️

Reply:
Python will first evaluate the list making it [('event_id', 'in', [1,2,3])] then convert the list to its representation making it "[('event_id', 'in', [1,2,3])]".

Aka
```
a = [2, 3, 4]
repr([('id', 'in', a)])
# return "[('id', 'in', [2, 3, 4])]"
```